Makefile

```diff
@@ -26,12 +26,13 @@ endif
 
 
 build: sbt
-	cd $(RESTSRV_DIR); \
+	cd $(RESTSRV_DIR) && \
 		$(SBT) ++$(SCALA_VERSION) -batch server/universal:stage
 
 clean: sbt
-	cd $(RESTSRV_DIR); \
+	cd $(RESTSRV_DIR) && \
 		$(SBT) ++$(SCALA_VERSION) -batch server/clean
+	find $(RESTSRV_DIR) -name "__pycache__" -exec rm -r {} \; || true
```

Comment:
changed `/bin/true` to which works on macOS and I think on all modern bash, at least I hope so.

Reply:
Well hard to say need to try, since make uses sh not bash by default...